src/status-codes.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import type {NotFoundStatus, OKStatus, RedirectStatus} from "./types.js";
+
+const NotFoundImpl: NotFoundStatus = {
+    get code(): 404 {
+        return 404;
+    },
+};
+
+/**
+ * Create a 404 NotFound status.
+ */
+export const notFound = (): NotFoundStatus => NotFoundImpl;
+
+const OKImpl: OKStatus = {
+    get code(): 200 {
+        return 200;
+    },
+};
+
+/**
+ * Create an 200 OK status.
+ */
+export const ok = (): OKStatus => OKImpl;
+
+/**
+ * Create a Redirect status.
+ *
+ * @param {string} targetURL The target URL of the redirect.
+ * @param {boolean} [isPermanent] If truthy, the status code 301 will be used to
+ * indicate a permanent move of the redirected URL; otherwise, the status code
+ * 302 will be used, indicating a temporary move. Defaults to false.
+ */
+export const redirect = (
```

Comment:
Hmm - I actually don't think this is how we want to implement this, I think this logic should be encapsulated inside the webapp rendering logic (namely that internally we'll know if we're redirecting, we'll return a status code of 301 or 302 and a correct Location header, but this system won't need to know that we "need to do a redirect" - it only needs to care about 3 things: status code, a list of headers, and the body text).